### ISSUE TITLE
Error building libsodium-sys

### ISSUE THREAD

**Issue by hubbabubbagump** [sodiumoxide/sodiumoxide#410]
When I try importing sodiumoxide the build crashes during `make check -j8`.

According to the test suite, the failing tests are:
```
FAIL: pwhash_argon2i
FAIL: pwhash_argon2id
```

Not too sure what's going on here, but I believe the command is seg faulting due to the errors:
```
../../build-aux/test-driver: line 107: 45211 Segmentation fault: 11  "$@" > $log_file 2>&1
../../build-aux/test-driver: line 107: 45222 Segmentation fault: 11  "$@" > $log_file 2>&1
```

Any help would be appreciated!

Output file for reference:
[output.txt](https://github.com/sodiumoxide/sodiumoxide/files/4039185/output.2.txt)

**Comment by kpp** [comment]
Would you please address this issue to https://github.com/jedisct1/libsodium/?

**Comment by hubbabubbagump** [comment]
Solved the issue.

The origin of this error was due to OSX Catalina and XCode 11. The good news is that this issue was fixed in XCode 11.2. To fix this, update to XCode 11.2+ and run the commands found [here](https://github.com/jedisct1/libsodium/issues/848#issuecomment-549163810):
```
sudo xcode-select -s /Applications/Xcode.app
sudo rm -rf /Library/Developer/CommandLineTools
```
